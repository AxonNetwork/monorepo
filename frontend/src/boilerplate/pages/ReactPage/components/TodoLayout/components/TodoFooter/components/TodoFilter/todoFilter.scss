@import 'sass/variables';

.todo-filter-btn {
  width: 120px;
  text-align: center;
  margin: 3px;
  padding: 0;
}

@media only screen and (max-width: $small-screen) {
  .todo-filter-btn {
    display: block;
    width: 80%;
    max-width: 300px;
    margin: auto;
  }
}
