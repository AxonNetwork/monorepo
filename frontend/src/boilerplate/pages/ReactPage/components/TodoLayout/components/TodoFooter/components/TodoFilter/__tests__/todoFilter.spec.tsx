import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';

import { TodoFilter } from '../todoFilter';

const dispatchMock = () => { return; };

describe('TodoFilter', () => {
  it('should renders correctly', () => {
    const renderer = TestRenderer.create(
      <TodoFilter active={true} setVisibilityFilter={dispatchMock}>
        {'test'}
      </TodoFilter>,
    );
    expect(renderer).toMatchSnapshot();
    renderer.unmount();
  });
});
