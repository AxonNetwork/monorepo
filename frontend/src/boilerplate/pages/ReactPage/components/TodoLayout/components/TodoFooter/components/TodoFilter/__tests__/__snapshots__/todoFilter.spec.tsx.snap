// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoFilter should renders correctly 1`] = `
<a
  className="btn waves-effect waves-light todo-filter-btn"
  href="#"
  onClick={[Function]}
>
  test
</a>
`;
